analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/PublicMethodArgumentsShouldBeCheckedForNull.CSharp9.cs

```diff
@@ -49,15 +49,16 @@ public void PatternMatching(object arg)
         }
     }
 
-    public void StaticLambda()
+    public void StaticLambda(object o)
     {
-        Func<object, string> a = static (arg) =>
-        {
-            return arg.ToString();      // Compliant, it's not a public method argument
-        };
-        a(null);
+        Func<object, string> lambda = static (object obj) => obj.ToString(); // Compliant - not accessible from other assemblies
```

Comment:
Here, we have parenthesized lambda. I'm missing simple lambda nearby
```
lambda = static obj => obj.ToString(); // Compliant - not accessible from other assemblies
```

Reply:
The method call after the parenthesized lambda [uses a simple lambda as a parameter](https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzguATEBqAPgAQCYBGAWACh8BmAAiJoGEaBvCm9u2/AFhoGUEEBFADGAGQhhgGCAAoA9sABWAUxEIa8gJRsOrch0N0ArAB5Fq9XDrEADAD4aAG0nSINALw2AbDQXK1DQstT0cLADoAFXkBACdYAHNZLQBuGgB6dMZ5MAAHJygIGA0AWhoYeQ0IEREVFBQoYCcVGgAzWJzNBAALFViaVBQVKQK63SNnVxkFVIpxowBZFR75DABBGpVchBR8Y2F5GFl8Yl8LUJZNZSiYhHiYJNSaAF9Zgw5nuff2XPiANyELR4NCWK3Wm22u32UEOx2IhHMAXUjmqBxgIWYLwozyAA===). Is there an easier way to create one?

Reply:
I'd like to have symmetrical way of testing it.
We have parenthesized tested for invoking the parenthesized.
We have simple tested for passing it as delegate.

That's what I find confusing and I actually missed the fact that the simple si present and tested there. Use the same approach for both and it should be fine.